xla/stream_executor/rocm/hip_blas_lt.cc

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "rocm/rocm_config.h"
+
+#if TF_HIPBLASLT
+#include "xla/stream_executor/rocm/hip_blas_lt.h"
+#include "../cuda/cuda_blas_lt.cc"
```

Comment:
This include is not ok. It is only allowed to include header files.

Reply:
This doesn't seem resolved yet. The linter complains that you are including a .cc file. It needs to be a .h file.

Reply:
Sorry, I forgot to add this.